### PR TITLE
Pass runtime worker configuration via env vars

### PR DESCRIPTION
**Pull request by telphan**

### CODE REVIEW

File: worker/lib/workers/qemu.ts [hunk 1]
```diff
@@ -418,27 +418,29 @@ class Qemu extends EventEmitter implements Leviathan.Worker {
 			this.references.network = undefined;
 		}
 
-		if (this.references.pool != null) {
-			await this.references.pool.stopAsync();
-			this.references.pool = undefined;
-		}
+		if (signal != null) {
+			if (signal == 'SIGTERM' || signal == 'SIGINT') {
```
Comment (roman-mazur) [inline worker/lib/workers/qemu.ts]: check for SIGTERM does not really make much sense: SIGTERM cannot be handled by the application; the OS kills the process immediately on this signal.

Comment (telphan) [reply worker/lib/workers/qemu.ts]: How do you mean SIGTERM cannot be handled by the application? I think the signal you are describing is SIGKILL.

File: worker/lib/helpers/index.ts [hunk 2]
```diff
@@ -155,3 +170,43 @@ export function resolveLocalTarget(target: string): PromiseLike<string> {
 		}
 	});
 }
+
+export async function getRuntimeConfiguration(
+	possibleWorkers: string[],
+): Promise<Leviathan.RuntimeConfiguration> {
+	const runtimeConfiguration: any = config.get('worker.runtimeConfiguration');
```
[the remark this answers is not in the document]
Comment (telphan) [reply worker/lib/helpers/index.ts]: The forEach lines below serve the purpose of detailed error messages when misconfiguration is passed.

File: worker/lib/workers/qemu.ts (hunk 1 above)
Comment (telphan) [reply worker/lib/workers/qemu.ts]: How do you mean SIGTERM cannot be handled? Afaik SIGTERM is the default way one should use to gracefully kill a process. 
I believe the signal you are describing is SIGKILL.

Comment (roman-mazur) [reply worker/lib/workers/qemu.ts]: yep, sorry, I confused myself

**Pull request merged**